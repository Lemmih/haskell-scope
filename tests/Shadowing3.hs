module Shadowing3 where

x x = x
  where
    x = 10
