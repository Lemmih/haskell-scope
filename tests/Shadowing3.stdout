Scope errors:

Definitions:
  Definition 1:
    [4mx x = x[0m
    [4m  where[0m
    [4m    x = 10[0m
  Definition 2:
    x [4mx[0m = x
  Definition 3:
        [4mx[0m = 10

Use sites:
  Definition used: 2
    [4mx[0m x = x
  Definition used: 3
    x x = [4mx[0m
